pkg/xds/generator/inbound_proxy_generator.go

```diff
@@ -32,7 +32,7 @@ func (g InboundProxyGenerator) Generate(ctx xds_context.Context, proxy *model.Pr
 
 		// generate CDS resource
 		localClusterName := envoy_names.GetLocalClusterName(endpoint.WorkloadPort)
-		clusterBuilder := envoy_clusters.NewClusterBuilder().
+		clusterBuilder := envoy_clusters.NewClusterBuilder(envoy_common.APIV2).
```

Comment:
Should you put a `todo` on these so we don't accidentally miss something when we get to configure the version?